docs/REST API/rest-api.md

```diff
@@ -6505,6 +6514,239 @@ Gets the result of a script that was executed.
 
 > Note: `exit_code` can be `null` if Fleet hasn't heard back from the host yet.
 
+### Upload a script
+
+_Available in Fleet Premium_
+
+Uploads a script, making it available to run on hosts of the specified team (or no team).
+
+`POST /api/v1/fleet/scripts`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| script          | file    | form | **Required**. The file containing the script.    |
+| team_id         | integer | form | The team id for the script. If specified, the script is applied to only hosts that are assigned to the specified team. If not specified, the profile is applied only to hosts not assigned to any team. |
```

Comment:
Took a stab at shortening this. Also, would use the phrase "no team" to be consistent with what we call things in the UI.
```suggestion
| team_id         | integer | form | The team ID. If specified, the script will only be available to hosts assigned to this team. If not specified, the script will only be available to hosts on **no team**.  |
```